integration-test/src/python/test_runner/main.py

```diff
@@ -13,9 +13,9 @@
 # The location of default configure file
 DEFAULT_TEST_CONF_FILE = "integration-test/src/python/test_runner/resources/test.conf"
 
-RETRY_ATTEMPTS = 15
+RETRY_ATTEMPTS = 25
 #seconds
-RETRY_INTERVAL = 30
+RETRY_INTERVAL = 10
```

Comment:
Waiting 30 secs for retry is too long, as most retry happens because posting result was slightly later than trying to fetch the result.